src/commissaire/storage/__init__.py

```diff
@@ -29,7 +31,10 @@ class StoreHandlerBase(object):
     Base class for all StoreHandler classes.
     """
 
-    # Subclasses override this, if applicable.
+    #: Class level LOGGER
+    logger = logging.getLogger('StorageHandler')
```

Comment:
That's a good idea. We are doing that else where so it would make sense to do it here too. Will add it in a fixup.